module/zfs/vdev_label.c

```diff
@@ -1449,8 +1496,9 @@ vdev_config_sync(vdev_t **svd, int svdcount, uint64_t txg)
 	 * then there's nothing to do.
 	 */
 	if (ub->ub_txg < txg &&
-	    uberblock_update(ub, spa->spa_root_vdev, txg) == B_FALSE &&
-	    list_is_empty(&spa->spa_config_dirty_list))
+	    uberblock_update(ub, spa->spa_root_vdev, txg,
+	    spa->spa_dsl_pool->dp_mmp.mmp_delay) ==
+	        B_FALSE && list_is_empty(&spa->spa_config_dirty_list))
```

Comment:
The indentation is wrong here.  Can we wrap this differently, or use a local variable to make this more readable.